### DISCUSSION TITLE
Server down alarm

### DISCUSSION THREAD

**Discussion by wolfgangyu** [syagent/agent-2#32] [General]
Thank you for using Syagent. I have been monitoring my server with Syagent for some time now and I am curious about how the server down alarm works. I noticed that the alarm is turned on and cannot be turned off in each server I monitored. However, when my host is disconnected, I can only log in to Syagent and notice that no monitoring data is being transmitted, but I have never received a server down alarm. Can you tell me under what conditions alarm will be triggered?

**Comment by Rashe** [comment]
> Thank you for using Syagent. I have been monitoring my server with Syagent for some time now and I am curious about how the server down alarm works. I noticed that the alarm is turned on and cannot be turned off in each server I monitored. However, when my host is disconnected, I can only log in to Syagent and notice that no monitoring data is being transmitted, but I have never received a server down alarm. Can you tell me under what conditions alarm will be triggered?

Hey, there is no big logic. If some monitored server did not get any data for some time (7 mins) it is marked as down, so you should get an alarm. Can you please provide me serverId so I can check why you didn't get the alarm :)

**Reply by wolfgangyu** [reply]
Please check this server for me: https://app.syagent.com/servers/64a379daff2f3a661ccfbcf5. This server frequently experiences disconnections due to power issues with the 5G router it uses. However, I have not received any server down alerts. I use the Pushover service.
![IMG_0011](https://github.com/syagent/agent-2/assets/49053016/83fb0e68-107a-4dfa-bedb-56681b2fd6d0)
However, resource alerts can be triggered, as shown in the above picture.

**Reply by Rashe** [reply]
How long is your downtime? Is it more than 7-10 mins?

**Comment by Rashe** [comment]
<img width="1295" alt="Screenshot 2024-01-04 at 16 45 34" src="https://github.com/syagent/agent-2/assets/1200792/0a84d6b3-5327-46ec-886a-0a0f198b1804">

Example of server id

**Comment by wolfgangyu** [comment]
Due to disconnections, the downtime often occurs during nighttime or holidays, frequently exceeding 7 minutes. Would you like me to disconnect again for your observation?Rafael Shemesh ***@***.***> 於 2024年1月6日 凌晨4:13 寫道：﻿
How long is your downtime? Is it more than 7-10 mins?

**Reply by Rashe** [reply]
Yes, if you can disconnect for 15 mins it will help me to understand.

**Reply by wolfgangyu** [reply]
I disconnect internet connection between 11:25 and 11:47 in UTC+8, and confirmed that I did not receive any notifications, including email and pushover.

**Reply by wolfgangyu** [reply]
When I found out that the internet was disconnected, the monitoring status was as shown in the figure below.
![image](https://github.com/syagent/agent-2/assets/49053016/f4431761-f824-490a-b240-d9510bfb03ba)

**Comment by Rashe** [comment]
There is a bug with it :)

Thx for report

**Comment by Rashe** [comment]
Fixed, thanks again for reporting

**Comment by wolfgangyu** [comment]
Great, do we need to update the agent then?Rafael Shemesh ***@***.***> 於 2024年1月11日 晚上10:11 寫道：﻿
Fixed, thanks again for reporting

**Reply by Rashe** [reply]
No, the agent itself is "stupid", the agent only collects server data and send it as is
